telco-examples/mgmt-cluster/airgap/eib/mgmt-cluster-airgap.yaml

```diff
@@ -27,136 +27,88 @@ kubernetes:
         createNamespace: true
         installationNamespace: kube-system
       - name: longhorn-crd
-        version: 104.2.0+up1.7.1
-        repositoryName: rancher-charts
+        version: 105.1.0+up1.7.2
+        repositoryName: rancher
         targetNamespace: longhorn-system
         createNamespace: true
         installationNamespace: kube-system
       - name: longhorn
-        version: 104.2.0+up1.7.1
-        repositoryName: rancher-charts
+        version: 105.1.0+up1.7.2
+        repositoryName: rancher
         targetNamespace: longhorn-system
         createNamespace: true
         installationNamespace: kube-system
       - name: metal3-chart
-        version: 0.8.1
+        version: 302.0.0+up0.9.0
         repositoryName: suse-edge-charts
         targetNamespace: metal3-system
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: metal3.yaml
       - name: neuvector-crd
-        version: 104.0.1+up2.7.9
-        repositoryName: rancher-charts
+        version: 105.0.0+up2.8.3
+        repositoryName: rancher
         targetNamespace: neuvector
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: neuvector.yaml
       - name: neuvector
-        version: 104.0.1+up2.7.9
-        repositoryName: rancher-charts
+        version: 105.0.0+up2.8.3
+        repositoryName: rancher
         targetNamespace: neuvector
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: neuvector.yaml
       - name: rancher
-        version: 2.9.1
+        version: 2.10.1
         repositoryName: rancher-prime
         targetNamespace: cattle-system
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: rancher.yaml
       - name: rancher-turtles-chart
-        version: 0.3.2
+        version: 302.0.0+up0.14.1
         repositoryName: suse-edge-charts
         targetNamespace: rancher-turtles-system
         createNamespace: true
         installationNamespace: kube-system
-        valuesFile: turtles.yaml
       - name: rancher-turtles-airgap-resources
-        version: 0.3.2
+        version: 302.0.0+up0.14.1
         repositoryName: suse-edge-charts
         targetNamespace: rancher-turtles-system
         createNamespace: true
         installationNamespace: kube-system
     repositories:
       - name: jetstack
         url: https://charts.jetstack.io
-      - name: rancher-charts
-        url: https://charts.rancher.io/
+      - name: rancher
+        url: https://charts.rancher.io
       - name: suse-edge-charts
-        url: oci://registry.suse.com/edge/3.1
+        url: oci://registry.suse.com/edge/3.2
       - name: rancher-prime
         url: https://charts.rancher.com/server-charts/prime
 embeddedArtifactRegistry:
   images:
```

Comment:
Rancher images are missing. Please use the ones listed [here](https://suse-edge.github.io/id-air-gapped-deployments-with-edge-image-builder.html#rancher-install).

Reply:
I will re-test it

Reply:
I've been testing it, and you don't need the full list you mentioned because with the current images listed in embedded and the images that hauler downloaded with EIB using the helm chart is enough for this use case. For example, I don't need calico for mgtm-cluster ... 
I've checked also with @e-minguez giving me more info to understand it. Thanks Edu!

Reply:
Just to double check, if you put charts (and/or K8s manifests) in the kubernetes.helm.charts EIB section, hauler will download both the charts and the images present there https://github.com/suse-edge/edge-image-builder/blob/ef4c06b8be34fd21138f0775cf608209296f16f8/docs/building-images.md?plain=1#L353-L356
The embeddedArtifactRegistry is for 'other' images you are planning to use or not the ones explicitly set in the charts.
Right?

Reply:
Yes, this is my understanding now...

Reply:
Yes, exactly. I'll approve so we can merge but let's test it together when we have the time, thanks!